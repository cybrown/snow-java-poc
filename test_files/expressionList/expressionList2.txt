if (1) 2 else 3;
def a = 3;
1+2;
a