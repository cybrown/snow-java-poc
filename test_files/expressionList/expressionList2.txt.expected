<ExpressionList>
    <ExpressionList>
        <ExpressionList>
            <If>
                <LiteralInteger>1</LiteralInteger>
                <LiteralInteger>2</LiteralInteger>
                <LiteralInteger>3</LiteralInteger>
            </If>
            <Definition>
                <Identifier>a</Identifier>
                <LiteralInteger>3</LiteralInteger>
            </Definition>
        </ExpressionList>
        <Binary operator="+">
            <LiteralInteger>1</LiteralInteger>
            <LiteralInteger>2</LiteralInteger>
        </Binary>
    </ExpressionList>
    <Identifier>a</Identifier>
</ExpressionList>
